src/components/interface/SwapForm/components/Leverage/Leverage.tsx

```diff
@@ -44,49 +44,15 @@ const Leverage = ({
     isUndefined(internalValue);
   const timer = useRef<number>();
 
-  const maxNotional = !isDisabled ? Math.min(notional, availableNotional) : 10;
-  const high = !isDisabled ? Math.max(maxNotional / margin, 1) : 20;
-  const low = 1;
-  const range = high - low;
-
-  const rainbow1 = !isDisabled ? Math.max(high / 2, 1) : 10;
-  const rainbow2 = !isDisabled ? Math.max(high / 1.5, 1) : 14;
-  const rainbow3 = !isDisabled ? Math.max(high / 1.2, 1) : 17;
-  const rainbow4 = !isDisabled ? Math.max(high / 1.05, 1) : 19;
-
-  const rainbow2Percent = (rainbow2 / high) * 100;
-  const rainbow3Percent = (rainbow3 / high) * 100;
-  const rainbow4Percent = (rainbow4 / high) * 10;
-
-  const rainbowStart = Math.max((1 - rainbow1 / range) * 100, 0);
-  const rainbowWidth = 100 - rainbowStart;
+  const options = [{ id: 100, text: "100x" }, { id: 500, text: "500x" }, { id: 1000, text: "1000x" }];
```

Comment:
perhaps better structure for the array is just array of numbers since the text is computed by appending 'x' to the number

Reply:
Turned it into just an array of numbers without text and id. 
Simpler